### PR TITLE
add manifest (fixes #129)

### LINKED ISSUE #129: Add a W3C Web-App Manifest

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -215,23 +220,22 @@
...
-    ga('create', 'UA-80530812-6', 'auto');
-    ga('send', 'pageview');
-  </script>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-35433268-69', 'auto');
```
Comment (fernandojsg) [inline index.html]: is this correct? we've a different analytics code now?

Comment (cvan) [reply index.html]: oh weird, I think my `git pull --rebase` automatically changed this, without any merge conflicts. good catch!

**Comment by cvan** [comment]
@fernandojsg updated

**Comment by fernandojsg** [comment]
cool thanks!

**Pull request merged**